dev/reference/tidb-operator-overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Overview of TiDB Operator
+summary: Learn the overview of TiDB Operator.
+category: reference
+---
+
+# Overview of TiDB Operator
+
+TiDB Operator is an automatic operation and maintenance system for TiDB cluster in Kubernetes that provides a full life-cycle management for TiDB including deployment, upgrades, expansion, backup, recovery, and configuration changes. With TiDB Operator, TiDB can run seamlessly in the Kubernetes clusters deployed on public or private cloud.
+
+## Architecture of TiDB Operator
+
+![TiDB Operator Overview](/media/tidb-operator-overview.png)
+
+TiDB Operator consists of the custom resources `TidbCluster`, the `tidb-controller-manager` components, the `tidb-scheduler` components and the command line tools `tkctl`:
+
+* `TidbCluster` is a custom resource defined using CRD (`CustomResourceDefinition`) to describe the expected state of the TiDB cluster;
+* `tidb-controller-manager` is a set of custom controllers in Kubernetes. These controllers constantly compare the expected state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. They adjust the resources in Kubernetes to drive the TiDB cluster to meet the desired state;
+* `tidb-scheduler` is an extension of the Kubernetes scheduler that injects the scheduling logic specific to TiBD clusters for the Kubernetes scheduler;
+* `tkctl` is the command line interface for the TiDB clusters in Kubernetes. It is used for issues related to the operation, maintenance and diagnosis of these clusters.
+
+![TiDB Operator Control Flow](/media/tidb-operator-control-flow.png)
+
+The diagram above is the analysis of the control flow of TiDB Operator. Because TiDB clusters also need components such as monitoring, initialization, scheduled backup, Binlog and so on, the definition of these clusters are encapsulated in a TiDB Operator using Helm Chart. The overall control process is as follows:
```

Comment:
"TiDB Operator encapsulates the definition of these components in the Helm chart"  maybe better